### PR TITLE
ROC-5502 Set paper response flag

### PR DESCRIPTION
**Pull request by karthikkannankainos**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ROC-5502

### Change description ###
Set paper response journey flag.

### CODE REVIEW

File: ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDScannedDocumentType.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package uk.gov.hmcts.cmc.ccd.domain;
+
+public enum CCDScannedDocumentType {
+    cherished,
+    other,
+    form,
+    coversheet
```
Comment (KiranVarma02) [inline ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDScannedDocumentType.java]: should these be in caps?

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ScannedDocument.java  1
- ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/ScannedDocumentMapperTest.java  1
- ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ScannedDocumentMapper.java  3
- domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ScannedDocumentType.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/cmc-claim-store/cmc-claim-store/pullRequest?prid=4458426&bid=15066072)

File: ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDScannedDocumentType.java (hunk 1 above)
Comment (karthikkannankainos) [reply ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDScannedDocumentType.java]: this is how the Bulk Scan sends entry to CCD, so we cant change it.

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ScannedDocumentMapper.java  3
- domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ScannedDocumentType.java  1
- domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ScannedDocument.java  1
- ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/ScannedDocumentMapperTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/cmc-claim-store/cmc-claim-store/pullRequest?prid=4458426&bid=15066072)

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ScannedDocument.java  1
- domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ScannedDocumentType.java  1
- ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ScannedDocumentMapper.java  3
- ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/ScannedDocumentMapperTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/cmc-claim-store/cmc-claim-store/pullRequest?prid=4458426&bid=15066072)

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ScannedDocumentType.java  1
- ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/ScannedDocumentMapperTest.java  1
- domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ScannedDocument.java  1
- ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ScannedDocumentMapper.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/cmc-claim-store/cmc-claim-store/pullRequest?prid=4458426&bid=15066072)

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ScannedDocumentType.java  1
- ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/ScannedDocumentMapperTest.java  1
- ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ScannedDocumentMapper.java  3
- domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ScannedDocument.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/cmc-claim-store/cmc-claim-store/pullRequest?prid=4458426&bid=15066072)

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ScannedDocument.java  1
- ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ScannedDocumentMapper.java  3
- domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ScannedDocumentType.java  1
- ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/ScannedDocumentMapperTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/cmc-claim-store/cmc-claim-store/pullRequest?prid=4458426&bid=15066072)

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ScannedDocument.java  1
- ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ClaimDocumentMapper.java  1
- ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ScannedDocumentMapper.java  3
- domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ScannedDocumentType.java  1
- ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/ScannedDocumentMapperTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/cmc-claim-store/cmc-claim-store/pullRequest?prid=4458426&bid=15066072)

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ScannedDocumentMapper.java  3
- domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ScannedDocument.java  1
- ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/ScannedDocumentMapperTest.java  1
- domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ScannedDocumentType.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/HMCTS/cmc-claim-store/pullRequest?prid=4458426&bid=15066072)

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ScannedDocumentMapper.java  3
- domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ScannedDocumentType.java  1
- ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/ScannedDocumentMapperTest.java  1
- domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ScannedDocument.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/HMCTS/cmc-claim-store/pullRequest?prid=4458426&bid=15066072)

**Pull request merged**